config/sidekiq_schedule_other.yml

```diff
@@ -0,0 +1,17 @@
+CacheRefreshSparkline:          # rake cache_refresh:sparkline
```

Comment:
What do you think about calling it `sidekiq_schedule_default.yml` instead of `sidekiq_schedule_other.yml`